tutorials/introductory/sample_plots.py

```diff
@@ -349,7 +383,7 @@
 
 .. _screenshots_eeg_demo:
 
-EEG demo
+EEG GUI
 =========
```

Comment:
Same as before: would it be possible to match the length of the '====' line with the line beyond?